src/main.cpp

```diff
@@ -4635,14 +4636,15 @@ void GridcoinServices()
 {
 
     //Dont do this on headless - SeP
-    #if defined(QT_GUI)
+    if(fQtActive)
+    {
        if ((nBestHeight % 125) == 0)
        {
             GetGlobalStatus();
             bForceUpdate=true;
             uiInterface.NotifyBlocksChanged();
```

Comment:
`uiInterface` is not available (iirc), so compiling this without Qt would fail even with gui. The preprocessor directives made it work.

Reply:
Hmm. Doesn't fail on a clean build with --disable-gui.

Reply:
Interesting... #include "ui_interface.h" is included and CClientUIInterface uiInterface is declared at the top of init.cpp. These are not conditional.